pkg/controller/node/nodecontroller.go

```diff
@@ -55,6 +55,7 @@ import (
 	utilversion "k8s.io/kubernetes/pkg/util/version"
 
 	"github.com/golang/glog"
+	"k8s.io/kubernetes/pkg/kubelet"
```

Comment:
I understand your motivation, but I'm not sure we necessarily want to have this package (which ends up part of kube-controller-manager) importing all of pkg/kubelet. cc @smarterclayton @derekwaynecarr